### DISCUSSION TITLE
Is this project forge 6 ready ?

### DISCUSSION THREAD

**Discussion by ClementTolois** [Deluze/electron-vue-template#35] [Q&A] [answered]
Hi, just a quick question about the recent release of forge 6. I am not very knowledgeable on the subject but I wanted to know if this project could be used effortlessly with forge 6? If not, are there any plans for it to become so?

**Comment by Deluze** [comment, marked as answer]
Forge is used to package & distribute Electron apps. To do this we are currently utilising [Electron Builder](https://www.electron.build/) so we would have to replace Builder with Forge and I don't have any plans for that (right now at least).

You should be able to get Forge to work with little effort. After all the code is transpiled (thanks to the `build.js` script) it lives in the `build` directory and we run Builder to package that directory into an app. Just a matter of replacing the `package.json` script to use Forge instead of Builder and tweak some config values.
